bindings/java/src/org/sleuthkit/datamodel/IngestModuleType.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Sleuth Kit Data Model
+ *
+ * Copyright 2011-2016 Basis Technology Corp.
+ * Contact: carrier <at> sleuthkit <dot> org
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *	 http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.sleuthkit.datamodel;
+
+/**
+ * Used to keep track of the module types
+ */
+public enum IngestModuleType {
+	DATA_SOURCE_LEVEL(0, "Data Source Level"), FILE_LEVEL(1, "File Level");
+	private final int typeID;
```

Comment:
Let's just use the enum value instead of a typeID field. We can also use name of the enum values for the typeName. There will be no need for getters for these. We can leave display names up to the UI.